lib/src/sql/array.rs

```diff
@@ -75,6 +74,16 @@ impl From<Array> for Vec<Value> {
 	}
 }
 
+impl FromIterator<Value> for Array {
+	fn from_iter<I: IntoIterator<Item = Value>>(iter: I) -> Self {
+		let mut a: Vec<Value> = vec![];
+		for v in iter {
+			a.push(v)
+		}
+		Array(a)
```

Comment:
Could be more efficient (collect to vec and convert vec to array) but I'm happy to PR this after the fact